Tests/StoreKitUnitTests/ProductsManagerTests.swift

```diff
@@ -46,13 +43,10 @@ class ProductsManagerTests: StoreKitConfigTestCase {
         let manager = try createManager(storeKit2Setting: .enabledForCompatibleDevices)
 
         let identifier = "com.revenuecat.monthly_4.99.1_week_intro"
-        var receivedProducts: Result<Set<StoreProduct>, PurchasesError>?
```

Comment:
No longer need to specify types every time, now they're inferred by the method we call.

Reply:
Never mind, that only works in Xcode 14.